Bitrise/testing_bitrise.yml

```diff
@@ -155,8 +157,10 @@ workflows:
         - is_compress: 'true'
         - debug_mode: 'true'
         - deploy_path: build/reports/
-    - cache-push:
-        run_if: true
+    - save-cache@1:
+        is_always_run: true
         inputs:
-        - cache_paths: |
-            .spm-build
+        - key: spm-cache-{{ checksum "Package.resolved" "*.xcodeproj/**/Package.resolved" }}
```

Comment:
Shouldn't the key be checksum include `"WeTransferPRLinter/Package.resolved"` - like in `l:84`?

Reply:
oh, yeah, you're totally right! It's a pitty we can't link these keys together more easily. Thanks for being sharp!